src/main/java/Event.java

```diff
@@ -0,0 +1,31 @@
+public class Event extends Task {
+    protected String from;
+    protected String to;
+
+    public Event(String description, String from, String to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+
+    public String getFrom() {
+        return from;
+    }
+
+    public void setFrom(String from) {
+        this.from = from;
+    }
+
+    public String getTo() {
+        return to;
+    }
+
+    public void setTo(String to) {
+        this.to = to;
+    }
+
+    @Override
+    public String toString() {
+        return "[E][" + getStatus() + "] " + description + " (from: " + from + " to: " + to + ")";
```

Comment:
Maybe can extract out the '[E]' to be a constant too, also applies for deadline/todo [D]/[T]